exec/java-exec/src/main/codegen/templates/Parser.jj

```diff
@@ -15,9 +15,11 @@
  * limitations under the License.
  */
 
-// TODO: Delete this file to reinstate its extraction from calcite-core.jar
-// once CALCITE-5579 is resolved and the incompatible grammar changes introduced
-// by CALCITE-5469 have been backed out. Also see: exec/java-exec/pom.xml.
```

Comment:
Do we want to leave the original info here just so that we know which Calcite PRs we're waiting for?